horovod/common/operations.cc

```diff
@@ -709,6 +698,27 @@ cudaError_t ReleaseCudaEvent(cudaEvent_t event) {
     }                                                                          \
   }
 
+int64_t TensorFusionThresholdBytes() {
```

Comment:
Let's rename to InitialTensorFusionThresholdBytes() to avoid confusion

Reply:
Sounds good.

Reply:
Nevermind -- I thought that this actually just read environment variables.  This looks good as-is.